### PR TITLE
Feature/healthcheck endpoint

### PR DESCRIPTION
**Pull request by JamesGardiner**

### CODE REVIEW

File: rabbit_monitor.py [hunk 1]
```diff
@@ -20,76 +19,121 @@
...
 
-        self.session = requests.Session()
-        self.session.auth = (self.settings.rabbit_default_user,
-                             self.settings.rabbit_default_pass)
 
-    def call_healthcheck(self):
-        logger.info('Getting rabbit healthcheck status')
-        healthcheck = self.session.get(self.urls['healthcheck'],
-                                       hooks=dict(response=self.process_healthcheck))
+@asyncio.coroutine
```
Comment (tundish) [inline rabbit_monitor.py]: Is this actually a coroutine? Looks like it returns a Task.

File: rabbit_monitor.py [hunk 2]
```diff
@@ -20,76 +19,121 @@
...
+                    rabbit_default_vhost=settings.RABBITMQ_DEFAULT_VHOST)
+
+healthcheck_url = settings.rabbit_url + 'healthchecks/node'
+aliveness_url = (settings.rabbit_url +
+                 'aliveness-test/{}'.format(settings.rabbit_default_vhost))
+
+urls = {'healthcheck': healthcheck_url,
+        'aliveness': aliveness_url}
+
+"""
```
Comment (tundish) [inline rabbit_monitor.py]: Remove inactive code.

File: rabbit_monitor.py [hunk 3]
```diff
@@ -20,76 +19,121 @@
...
+def shutdown():
+    logger.info("Shutting down sdx-rabbit-monitor")
+    sys.exit()
+
+"""
+
+
```
Comment (tundish) [inline rabbit_monitor.py]: Maybe group the coroutines under a Monitor class so it's clear which are related.

File: rabbit_monitor.py [hunk 4]
```diff
@@ -1,13 +1,12 @@
...
+import os
 
-import requests
-from requests.adapters import HTTPAdapter
-from requests.packages.urllib3.util.retry import Retry
+import aiohttp
+from aiohttp import web
 from structlog import wrap_logger
 
 import settings
```
Comment (tundish) [inline rabbit_monitor.py]: Could we do:
`import settings as env`

to differentiate between values from settings module (which come straight from os.env) and the settings data structure.

**Review by tundish — DISMISSED** [review]

File: rabbit_monitor.py (hunk 3 above)
Comment (JamesGardiner) [reply rabbit_monitor.py]: In the interest of getting this done I'm going to skip this and add a potential issue

**Comment by JamesGardiner** [comment]
@necrophonic - can you ok this for merge?

**Comment by necrophonic** [comment]
Should probably document the available endpoints (healthcheck) in the readme

**Comment by JamesGardiner** [comment]
@necrophonic done :+1:

**Review by necrophonic — APPROVED** [review]

**Pull request merged**